includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature

```diff
@@ -3,17 +3,15 @@ Feature: Mayfield Homepage Layout
   As an end user
   I want to check for the existence of content that should appear
 
-  @api @javascript
+  @api @javascript @jsv
```

Comment:
Probably should add a @dev and @destructive to this as it does configuration change.